File: JS/31-40/e_32.js
// Genera un número aleatorio entre 25 y 50 (ambos inclusive)
let numeroAleatorio = Math.floor(Math.random() * (50 - 25 + 1)) + 25;

// Imprime el número generado
console.log("Número aleatorio entre 25 y 50:", numeroAleatorio);


// En JavaScript, puedes usar la función Math.random() para generar un número decimal aleatorio entre 0 y 1, y luego ajustar ese número al rango deseado.
// En este ejemplo, Math.random() genera un número decimal aleatorio entre 0 (inclusive) y 1 (exclusivo).
// Luego, multiplicamos ese número por la diferencia entre los límites superior e inferior del rango deseado,
// le sumamos el límite inferior y aplicamos Math.floor() para redondear hacia abajo y obtener un número entero.
// Esto asegura que obtendrás un número en el rango de 25 a 50 (ambos inclusive).

File: Java/src/e31/e_32.java
package e31;

import java.util.Random;

public class e_32 {
    public static void main(String[] args) {
        // Crea una instancia de la clase Random
        Random random = new Random();

        // Genera un número aleatorio entre 25 y 50 (ambos inclusive)
        int numeroAleatorio = random.nextInt(26) + 25;

        // Imprime el número generado
        System.out.println("Número aleatorio entre 25 y 50: " + numeroAleatorio);
    }
}

// En Java, puedes usar la clase Random para generar un número aleatorio y la
// fórmula adecuada para ajustarlo al rango deseado. En este ejemplo,
// random.nextInt(26) genera un número aleatorio entre 0 (inclusive) y 25
// (exclusivo), y luego le sumamos 25 para ajustar el rango al intervalo
// deseado. Al ejecutar este programa, obtendrás un número aleatorio en el rango
// de 25 a 50 (ambos inclusive).